src/marqo/connections.py

```diff
@@ -56,7 +56,7 @@ def init_from_app(self, host, port):
             logger.info(f"Took {((t1-t0)*1000):.3f}ms to connect to redis and load scripts.")
 
         except Exception as e:
-            logger.warn(f"There is a problem with your redis instance. Could not load throttling scripts onto redis. Reason: {e}")
+            logger.warn(f"There is a problem with your redis connection. Could not load throttling scripts onto redis. To suppress these warnings, disable throttling with export MARQO_ENABLE_THROTTLING='FALSE'. Read more under Redis setup section of the developer guide: https://github.com/marqo-ai/marqo/tree/mainline/src/marqo#developer-guide. Redis error reason: {e}")

```

Comment:
nit enclosing code within backticks helps readability: 
```
`export MARQO_ENABLE_THROTTLING='FALSE'` 
```

Reply:
Does this change the formatting in any way? Or is it just so they see the backticks?